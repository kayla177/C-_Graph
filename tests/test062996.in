ENTITY X745BX4T name_X745 journal
ENTITY N2XYORU9 name_N2XY institution
ENTITY N7HD948D name_N7HD institution
RELATIONSHIP X745BX4T cited_by N2XYORU9 8.5
PRINT X745BX4T
HIGHEST
EXIT
