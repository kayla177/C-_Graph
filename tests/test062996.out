success
success
success
success
N2XYORU9
X745BX4T N2XYORU9 8.5
